corehq/apps/reports/tests/test_views.py

```diff
@@ -76,6 +76,11 @@ def test_when_user_cannot_view_report_receives_404(self):
         with self.assertRaises(Http404):
             self.email_report(report_name='worker_activity')
 
+    def test_disallowed_request_get_method(self):
+        self.request = self._create_request(method='get')
+        response = self.email_report()
+        self.assertEqual(response.status_code, 405)
```

Comment:
I see that in all instances where we check `response.status_code` we always use the status code directly (i.e. the number), including when this is `405`. It's not that common and that's why I mentioned its meaning in the PR, but you're right that we could do better in the test code.
I would still like for this to be consistent across our codebase, so what about a comment? Or we can make a PR to change the ones for which there is such a specific class.

Reply:
I count 200+ occurrences, with 75 for error codes.
I'll start by changing the `405` instances, and we can continue the discussion in a new PR.

Reply:
I see what you mean. Totally up to you. I view this as very similarly wanting to start using `let` in Javascript rather than `var`. Yes, the bulk of our codebase is still using `var`, but it's just an objectively worse way of doing things. I wouldn't object to just start using `let` rather than require an entire migration of all existing `var` instances